README.md

```diff
@@ -17,8 +17,16 @@ To view the site locally install Jekyll:
 
     gem install jekyll
 
-start the server:
+Install Bundler:
 
-    jekyll serve
+    gem install bundler
+
+Install Jekyll and other dependencies from the GitHub Pages gem:
+
+    bundle install
```

Comment:
Before you can do this and even before you can run `gem` you need the following packages (on Debian at least): ruby ruby-dev zlib1g-dev.

Could you add a paragraph and a command with instructions to install these?